Documentation/running.md

```diff
@@ -46,6 +46,7 @@ FLANNEL_SUBNET=10.5.72.1/24
 FLANNEL_MTU=1450
 FLANNEL_IPMASQ=false
 ```
+The `FLANNEL_SUBNET` value written in this subnet config file will be checked each time flannel starts and will be reused if the subnet still matches the values stored in the etcd config.
```

Comment:
This is a bit ambiguous so will be confusing for uses. Can you make it clearer and more explicit?